locales/pt_BR/pcsx2_Main.po

```diff
@@ -1072,8 +1068,7 @@ msgid "Create a new memory card"
 msgstr "Criar um novo cartão de memória"
 
 #: pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp:59
-#: pcsx2/gui/Panels/DirPickerPanel.cpp:73
-#: pcsx2/gui/Panels/DirPickerPanel.cpp:271
+#: pcsx2/gui/Panels/DirPickerPanel.cpp:73 pcsx2/gui/Panels/DirPickerPanel.cpp:271
```

Comment:
Why did you change this part ? It was fine before.

Reply:
Poedit does this.